docs/pages/agents/build-an-agent.mdx

```diff
@@ -191,3 +222,28 @@ Consider registering an [ENS domain](https://ens.domains/) for your agent to mak
 ### Example Railway deployment
 
 For reference, here's an example Railway deployment of a [gm-bot](https://railway.com/deploy/UCyz5b) agent.
+
+## Monitor and maintain your agent
+
+### Monitor agent health
+
+```tsx
+const inboxState = await client.preferences.inboxState();
+const address = inboxState.identifiers[0].identifier;
+const inboxId = client.inboxId;
+const installationId = client.installationId;
+const conversations = await client.conversations.list();
+
+console.log(`
+XMTP Client:
+  • InboxId: ${inboxId}
+  • Address: ${address}
+  • Conversations: ${conversations.length}
+  • Installations: ${inboxState.installations.length}
+  • InstallationId: ${installationId}
+  • Network: ${process.env.XMTP_ENV}
+`);
+```
+
+### Log agent activity
```

Comment:
Bullet from Base docs:  Logging: Add comprehensive logging to track agent activity

Do we want to say more?